_includes/pbc/all/install-features/202311.0/install-the-service-points-cart-feature.md

```diff
@@ -44,16 +46,79 @@ service_point_cart.checkout.validation.error,Der ausgewählte Servicepunkt "%uui
+| ServicePointCheckoutPreConditionPlugin         | Validates if `QuoteTransfer.items.servicePoint` is active and available for the current store. |           | Spryker\Zed\ServicePointCart\Communication\Plugin\Checkout                |
+| ReplaceServicePointQuoteItemsQuoteMapperPlugin | If shipments are provided, replaces quote items using an applicable strategy.                      |           | Spryker\Zed\ServicePointCartsRestApi\Communication\Plugin\CheckoutRestApi |
+
+**src/Pyz/Zed/Checkout/CheckoutDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Checkout;
+
+use Spryker\Zed\Checkout\CheckoutDependencyProvider as SprykerCheckoutDependencyProvider;
+use Spryker\Zed\ServicePointCart\Communication\Plugin\Checkout\ServicePointCheckoutPreConditionPlugin;
+
+class CheckoutDependencyProvider extends SprykerCheckoutDependencyProvider
+{
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return array<\Spryker\Zed\CheckoutExtension\Dependency\Plugin\CheckoutPreConditionPluginInterface>
+     */
+    protected function getCheckoutPreConditions(Container $container): array
+    {
+        return [
+            new ServicePointCheckoutPreConditionPlugin(),
+        ];
+    }
+```
 
 {% info_block warningBox "Verification" %}
 
 1. Add an item to cart and proceed to checkout.
 2. Select a service point.
 3. Deactivate the service point.
-4. Proceed to the *Summary* page.
-    On the *Summary* page, make sure you get the validation error.
+4. Proceed to the **Summary** page.
+    On the **Summary** page, make sure you get the validation error.
+
+{% endinfo_block %}
+
+**src/Pyz/Zed/CheckoutRestApi/CheckoutRestApiDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\CheckoutRestApi;
+
+use Spryker\Zed\CheckoutRestApi\CheckoutRestApiDependencyProvider as SprykerCheckoutRestApiDependencyProvider;
+use Spryker\Zed\ServicePointCartsRestApi\Communication\Plugin\CheckoutRestApi\ReplaceServicePointQuoteItemsQuoteMapperPlugin;
+
+class CheckoutRestApiDependencyProvider extends SprykerCheckoutRestApiDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Zed\CheckoutRestApiExtension\Dependency\Plugin\QuoteMapperPluginInterface>
+     */
+    protected function getQuoteMapperPlugins(): array
+    {
+        return [
+            new ReplaceServicePointQuoteItemsQuoteMapperPlugin() # Has to be placed before PaymentsQuoteMapperPlugin
+        ];
+    }
+```
+
+{% info_block warningBox "Verification" %}
+
+1. Prepare two product offers for the same product:
+  1. With support for the pickup shipment type and a connection to a service point.
+  2. Without support for the pickup shipment type.
+2. Add the product offer 2 to cart.
+3. Proceed to the `checkout-data` resource in the GLUE API.
```

Comment:
What do you mean by `proceed` in this case?

Reply:
Navigate to the "checkout-data" resource or the next request has to be done towards this resource.